pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -162,13 +162,22 @@ func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
 	return err
 }
 
-func (h *NetworkUtilsHandler) IsIpv6Enabled() bool {
-	podIp := os.Getenv("MY_POD_IP")
-	if !netutils.IsIPv6String(podIp) {
-		log.Log.V(5).Info("Since the pod ip is non IPv6, IPv6 is disabled")
+func (h *NetworkUtilsHandler) IsIpv6Enabled(interfaceName string) bool {
+	link, err := Handler.LinkByName(interfaceName)
+	addrList, err := Handler.AddrList(link, netlink.FAMILY_ALL)
```

Comment:
We could request the `FAMILY` to be IPv6 and then drop one of the checks in the following code. Any reason not to?

Reply:
No reason, I will.do it. Thanks.